package.json

```diff
@@ -19,31 +19,84 @@
     "integration-tests:alpha": "node build/testing/sauce-labs/test-browsers.js alpha",
     "integration-tests:beta": "node build/testing/sauce-labs/test-browsers.js beta",
     "integration-tests:release": "node build/testing/sauce-labs/test-browsers.js release",
-    "prettier:diff": "prettier --config .prettierrc '**/*.{ts,tsx,js}' --list-different"
+    "prettier:diff": "prettier --config .prettierrc '**/*.{ts,tsx,js}' --list-different",
+    "test": "npm run tslint && npm run unit-tests",
+    "tslint": "tslint -c ./tslint.json '*.{ts,tsx,js}'",
+    "tslint:fix": "tslint -c ./tslint.json --fix '*.{ts,tsx,js}'",
+    "unit-tests": "jest --maxWorkers=4",
+    "unit-tests:watch": "jest --watch",
+    "watch": "tsc -p ./tsconfig.json -w --preserveWatchOutput"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "build/helpers/*.ts"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "statements": 100,
+        "branches": 100,
+        "functions": 100,
+        "lines": 100
+      }
+    },
+    "testURL": "http://localhost",
+    "transform": {
+      "^.+\\.ts?$": "ts-jest",
+      "^.+\\.js?$": "babel-jest"
+    },
+    "transformIgnorePatterns": [
+      "!<rootDir>/node_modules/lodash-es",
+      "node_modules/jss-*",
+      "node_modules/css-vendor"
+    ],
+    "testPathIgnorePatterns": [
+      "packages/",
+      "build/helpers/__tests__/*"
+    ],
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(js?|ts?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "js"
+    ],
+    "moduleDirectories": [
+      "node_modules"
+    ]
+  },
   "lint-staged": {
-    "*.{ts,tsx,js}": [
```

Comment:
Got it

Reply:
This is true, with exception being that lint-staged runs at the git root. This will *not* lint .tsx files staged for commit and run prettier. We should add this back in.